src/tronls.jl

```diff
@@ -1,26 +1,9 @@
-export TronSolverNLS
+export TronSolverNLS, SolverTools
```

Comment:
```suggestion
export TronSolverNLS
```
You don't need this as there is a `using SolverTools` in the package already.